forem/forem
# ✂️✂️✂️: Remove unused partial `menu`

## What type of PR is this? (check all applicable)


## Description
In this [PR](https://github.com/forem/forem/pull/17388), we removed `<%= render "menu" %>` which was the old Member Index Menu. This PR removes the partial as it's no longer being used anywhere. 

## Related Tickets & Documents



- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings

1. Click around `/admin/users`
2. Double check for references to the `menu` partial. 

### UI accessibility concerns?
None.

## Added/updated tests?

- [x] No, because we only remove an unused partial 
      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/SvRDkFda8jWjUanyKt/giphy.gif)
